src/components/SideBar/SideBarStyled.js

```diff
@@ -100,3 +101,19 @@ export const ChatsButtonsFrame = styled.div`
   width: 51px;
   height: 166px;
 `;
+
+export const UnreadMessagesCount = styled.div`
```

Comment:
Isn't should be span or p tag instead of div?

Reply:
done